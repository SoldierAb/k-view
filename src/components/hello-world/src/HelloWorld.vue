<template>
  <div class="hello">
    <h1>{{ msg }}</h1>
  </div>
</template>

<style lang="less">
  .hello{
    color:@font-color;   /** 默认皮肤  src\theme\default.less */ 
    font-size: @font-size-base;  /** 通用基础字体  src\theme\common.less */
  }
</style>

<script>
export default {
  name: 'HelloWorld',  // 需要加上 V 前缀，统一使用大写驼峰形式
  data() {
    return {
      msg: 'Hello World',
    };
  },
};
</script>

