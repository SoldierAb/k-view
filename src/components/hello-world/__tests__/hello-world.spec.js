import { shallowMount } from '@vue/test-utils';
import HelloWorld from '../index';

describe('<v-hello-world/>', () => {
  it('should render correct contents', () => {
    const wrapper = shallowMount(HelloWorld);
    // expect(wrapper.find('.hello h1').text())
    //   .toEqual('Hello Jest');
      expect(wrapper.find('.hello h1').text())
      .toMatchSnapshot();
  });
});