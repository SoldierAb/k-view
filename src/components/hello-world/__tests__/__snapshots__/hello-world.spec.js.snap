// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<v-hello-world/> should render correct contents 1`] = `"Hello World"`;
